### ISSUE TITLE
Initial 'Footer' section Figma design

### ISSUE THREAD

**Issue by mehulkapoor020** [mehulkapoor020/IS601-EndTerm#14]
Summary:
Design the 'Footer' section for 'Le Jardin' website on Figma. Footer section should have all the important links that the cutomer would want to access.

Acceptance criteria:
1.Follow the Web Content Accessibility Guidelines (WCAG) to make the website accessible to all.
2 Add links to About section, Review section, Menu section and Home page.
3.Add name of the bakery in the footer.
4.Use multiple methods of persuasions to attract cutomers for this website.
2.Use Component Library like MUI and NextUI. httphttps://mui.com/material-ui/react-bottom-navigation/
3.Adhere to the Brand Archetype persona 'Explorer'.
4.Use Adobe color wheel for color selection of the Website components.

**Comment by mehulkapoor020** [comment]
<img width="607" alt="Screenshot 2023-12-12 at 11 44 20 PM" src="https://github.com/mehulkapoor020/IS601-EndTerm/assets/144635746/ff1e11ca-5527-4269-a0ed-221f05c07ade">


Footer section design figma https://www.figma.com/file/puog0pDY0mW5VGzuivGCi1/Final-Group-Project--Bakery?type=design&node-id=0%3A1&mode=design&t=QsCIpjv2i3C0tP7D-1
